docs/smart-contracts/multisig.md

```diff
@@ -35,32 +160,35 @@ proposal_type: type = sp.big_map[
 
 The `submit_proposal` entrypoint allows authorized users to submit a payment amount and an account address, which adds a proposal to the storage:
 
-```python
+```smartpy
+startingVoters = sp.set()
+startingVoters.add(sp.sender)
 self.data.proposals[self.data.activeProposalId] = sp.record(
     paymentAmt=params.paymentAmt,
     receiver=params.receiver,
-    voters=sp.set(sp.sender),
+    voters=startingVoters,
```

Comment:
is this just for clarity?

Reply:
Yea, that's probably the best way -- I must have blanked on the `{}` syntax.